reduction/lr_reduction/reduction_template_reader.py

```diff
@@ -48,8 +48,8 @@ def __init__(self):
         self.norm_file = 0
 
         # Clean up options: cut first and last points as needed
```

Comment:
this comment is not necessary anymore now, right? And why are pre and post cuts unnecessary now?